app/models/post.rb

```diff
@@ -1,3 +1,14 @@
 class Post < ApplicationRecord
-  belongs_to :Author_id
+  belongs_to :Author, class_name: 'User'
+  has_many :likes
+
+  after_save :update_posts_counter
+
+  def update_posts_counter
+    Author.increment!(:Posts_Counter)
```

Comment:
- [ ] As I mentioned before name in the convention is very important, here you are increasing  the `:Posts_Counter` column which does not exist in the User table instead the User table contains the `:PostsCounter`